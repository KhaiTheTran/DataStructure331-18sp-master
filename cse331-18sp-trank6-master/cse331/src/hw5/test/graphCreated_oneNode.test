#One node test

# Create a graph
CreateGraph graphCreated_oneNode

# Add a node
AddNode graphCreated_oneNode xx

# Print the nodes in the graph and the outgoing edges from xx
ListNodes graphCreated_oneNode
ListChildren graphCreated_oneNode xx