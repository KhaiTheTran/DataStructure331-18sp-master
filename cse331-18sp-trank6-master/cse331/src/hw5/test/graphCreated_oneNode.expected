#One node test

# Create a graph
created graph graphCreated_oneNode

# Add a node
added node xx to graphCreated_oneNode

# Print the nodes in the graph and the outgoing edges from xx
graphCreated_oneNode contains: xx
the children of xx in graphCreated_oneNode are: